### DISCUSSION TITLE
Support for MariaDB

### DISCUSSION THREAD

**Discussion by NA0341** [go-shiori/shiori#533] [Feature requests]
Since it already supports MySQL - for efficient server deployments it would make sense to use it with MariaDB.

**Comment by fmartingr** [comment]
You should be able to use MariaDB directly with the MySQL engine. Our test suite [uses MariaDB on the CI](https://github.com/go-shiori/shiori/blob/master/.github/workflows/_test.yml#L24) to run the database tests.
